docs/organizing-and-exploring-results-in-the-ui/searching-experiments.rst

```diff
@@ -1,17 +1,31 @@
+Searching and filtering experiments
+===================================
+
+Basic Filtering
+---------------
+You can perform the simplest filtering by typing into the search fields:
+
+   .. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/search_fields.png
+      :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/search_fields.png
+      :alt: Experiment search
+
+
+For more advanced filtering criteria you should switch to the "advanced search" and use Neptune Query Language described in the next section.
```

Comment:
[opinion] I'm afraid that reader may not discover how to do it. Maybe we can provide some help here?